# PUMP_STATUS message data

Header header

float32 pump_speed
float32 spray_speed